### PR TITLE
Merge Squeezed into xenopsd

### PR DESCRIPTION
**Pull request by psafont**
Also upgrades the dune metadata used in squeezed to dune 2.0.

git subtree add --prefix sqeezed https://github.com/xapi-project/squeezed master the cleanups start at commit 9ea7bf38f5250aa94a40f8d9f0b93bede18b069f

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,6 +44,7 @@ install:
 	install -D ./scripts/common.py $(DESTDIR)/$(LIBEXECDIR)/common.py
 	install -D ./scripts/igmp_query_injector.py $(DESTDIR)/$(LIBEXECDIR)/igmp_query_injector.py
 	install -D ./scripts/qemu-wrapper $(DESTDIR)/$(QEMU_WRAPPER_DIR)/qemu-wrapper
+	install -D _build/default/squeezed/src/squeezed.exe $(DESTDIR)/$(SBINDIR)/squeezed
```
Comment (robhoes) [inline Makefile:47]: I think it should come from `_build/install/default/bin`.

File: README.md [hunk 2]
```diff
@@ -19,3 +19,35 @@ This code can be profiled for coverage. See [COVERAGE.md].
...
+Squeezed uses [ballooning](http://static.usenix.org/events/osdi02/tech/full_papers/waldspurger/waldspurger_html/node6.html)
+to move memory between running VMs. It is able to:
+
+  1. avoid wasting host memory: unused memory can be gifted to VMs
+  2. share memory according to a configured policy, so some VMs will use more than others
+  3. "squeeze" existing VMs to make room to start new VMs.
+
+Squeezed is an optional component of the [xapi toolstack](http://wiki.xen.org/wiki/Choice_of_Toolstacks).
+
+building the development version
```
Comment (robhoes) [inline README.md:34]: Let's delete these instructions, which aren't right anymore.

**Review by lindig — APPROVED** [review]
I am just looking at the changes on top of the merge of the two repos and this looks good to me.

**Review by edwintorok — APPROVED** [review]
We can probably further cleanup the Makefile, but that can wait until we merge more of these repos (see my changes on message-switch that move most of the installation into Dune).

File: README.md (hunk 2 above)
Comment (robhoes) [reply README.md:34]: These incorrect instructions are still there.

Comment (psafont) [reply README.md:34]: documentation about coverage is also out of date

**Comment by robhoes** [comment]
There are several dot files in `squeezed/` that should also be deleted (e.g. travis config and git files).

**Review by robhoes — APPROVED** [review]

**Review by lippirk — APPROVED** [review]

**Comment by lindig** [comment]
I think we should merge this before it gets outdated. It will require corresponding changes in spec files.

**Pull request merged**